lib/cancel/Cancel.ts

```diff
@@ -0,0 +1,13 @@
+export class Cancel {
```

Comment:
As this and some other parts are adapted from axios, I think we have to attribute it here

Reply:
That sounds proper, yes.

Reply:
Fixed in https://github.com/VanTanev/jest-mock-axios/commit/fc6abf09b3a4d0da36e81ddd05b467b9a2638241